advocacy_docs/supported-open-source/patroni/migrating.mdx

```diff
@@ -0,0 +1,323 @@
+
+1. `postgres-node-2` has received all WALs between the LSNs reported by these replication slots in `postgres-node-1`:
+    1. `slot_for_postgres_node_2` &mdash; previously existing slot in the original cluster.
+    2. `postgres_node_2` &mdash; slot automatically created by Patroni based on node name.
+2. `postgres-node-3` has received all WALs between the LSNs reported by these replication slots in `postgres-node-1`:
+    1. `slot_for_postgres_node_3` &mdash; previously existing slot in the original cluster.
+    2. `postgres_node_3` &mdash; slot automatically created by Patroni based on node name.
+
+**Note:** You can use a query like the one below to check the required LSN of these slots. You need to make sure that `restart_lsn` of the slots created by Patroni are `>=` the `restart_lsn` of the original slots. On node `postgres-node-1`, through a `psql` session:
+
+```sql
+SELECT slot_name,
+       restart_lsn
+FROM pg_catalog.pg_replication_slots
+WHERE slot_name IN (
+    'slot_for_postgres_node_2',
+    'postgres_node_2',
+    'slot_for_postgres_node_3',
+    'postgres_node_3'
+)
+```
+
+Once the above conditions are met we are then able to drop the original slots (`slot_for_postgres_node_2`, and `slot_for_postgres_node_3`). In order to do that, assuming you have not configured any other permanent slots other than `slot_for_postgres_node_2` and `slot_for_postgres_node_3`, follow these steps:
+
+1. Remove [permanent slots](https://patroni.readthedocs.io/en/latest/dynamic_configuration.html#:~:text=a%20standby%20leader-,slots,-%3A%20define%20permanent%20replication) dynamic configuration. On any Postgres node:
+
+```bash
+patronictl -c /etc/patroni.yml edit-config test_cluster --set "slots=null" --force
+```
+
+2. Wait for about 10 seconds and check that replication slots `slot_for_postgres_node_2` and `slot_for_postgres_node_3` had been dropped from `postgres-node-1` -- query `pg_replication_slots` to check that.
+
+#### 5. Test switchover and switchback
+
+At this point your cluster is completely managed by Patroni, so it is important to perform a switchover and switchback operation during a maintenance window to ensure Patroni is operating correctly.
+
+As mentioned, you should perform the following steps during a maintenance window. As an example we will perform a switchover and switchback between `postgres-node-1` and `postgres-node-2`. The steps:
+
+1. Check replication lag between nodes. On any Postgres node:
+
+```bash
+patronictl -c /etc/patroni.yml list test_cluster
+```
+
+**Note:** the output should show value `0` for column `Lag in MB`, for rows `postgres-node-2` and `postgres-node-3`. Do not proceed with the next steps if that is not the case.
```

Comment:
I suggest pasting an example output here

Reply:
Sounds good, will work on that.

Reply:
Done!